libraries/AP_RangeFinder/RangeFinder.h

```diff
@@ -21,7 +21,7 @@
 #include <AP_SerialManager/AP_SerialManager.h>
 
 // Maximum number of range finder instances available on this platform
-#define RANGEFINDER_MAX_INSTANCES 2
+#define RANGEFINDER_MAX_INSTANCES 4
```

Comment:
I would like to install the device in 10 directions of "0: Forward 1: Forward-Right 2: Right 3: Back-Right 4: Back 5: Back-Left 6: Left 7: Forward-Left 24: Up 25: Down"

Reply:
@muramura, I think that at some point it becomes too many parameters and an external computer should be used to consolidate the readings from many lidar.